src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -617,7 +617,8 @@ window.qBittorrent.DynamicTable = (function() {
         onSelectedRowChanged: function() {},
 
         updateRowData: function(data) {
-            const rowId = data['rowId'];
+            // ensure rowId is a string
```

Comment:
Just curious, isn't `rowId` supposed to be a number? or it could be other types?
Would it be hard to enforce it as a number ?

Reply:
Most tables set the `rowId` to a string value, like the torrent hash or tracker url. Some newer tables (search, files, rss) instead use a numeric array index. I remember adding the initial numeric `rowId` implementation with the search table, and wondering at the time if the type change was a bad idea. It took a while, but turns out it was.